crates/autopilot/src/run.rs

```diff
@@ -624,10 +624,11 @@ pub async fn run(args: Arguments) {
     let market_makable_token_list =
         AutoUpdatingTokenList::from_configuration(market_makable_token_list_configuration).await;
 
+    let db = Arc::new(db);
     let run = RunLoop {
         eth,
         solvable_orders_cache,
-        database: Arc::new(db),
+        database: db.clone(),
```

Comment:
can we simply wrap the other database methods inside persistence to have the run loop only interface with one persistence infra module?

Reply:
yes, removed database from `RunLoop`